planning/testnet-kpis.md

```diff
@@ -1,29 +1,40 @@
+The testnet success critera have been expressed below in the form of Key Performance Indicators with the rationale for their inclusion, the data source, the actual metric and the target value.
+
+## Testnet KPIs
 Key performance indicators (KPIs) will be used to determine the amount of Testnet traction over time.
 
-## Selected KPIs
+### Selected KPIs for Users
 | KPI NAME | RATIONALE | SOURCE | METRIC | TARGET |
 |--|--|--|--|--|
-| Testnet documentation page views (https://docs.obscu.ro/testnet/) | Gauges interest from developers in building on testnet. A good early indicator. Straightforward to capture | Google Analytics | Number of unique page views in the last 4 weeks | Targets will be determined once initial baseline data is captured |
-| Testnet uptime | Captures how robust and ready for mainnet Obscuro is | DataDog avg:system.uptime{*} | Average Testnet uptime over the last 4 weeks||
-| Number of end user accounts| Captures users using testnet in general and testing new dApps and shows the pipeline from wallet extension download to actively using testnet | TBD | Number of new end user account addresses in the last 4 weeks ||
-|DApps in development| This is difficult to measure as it’s not something we can look to the network to accurately measure. We can use our business development CRM data as an indicator. | Airtable "Partnerships" tab| Number of partners with status changed to "Soft Commitment" in the last 4 weeks||
-| New dApps deployed on Testnet| Indicates how much new activity and commitment of effort from dApp builders| ObscuroScan?| Number of new dApp addresses deployed in the last 4 weeks||
-| Wallet extension downloads| Good proxy for the number of active users. Straightforward to capture.| GitHub|Number of wallet extension downloads in the last 4 weeks||
+| Testnet uptime | Captures how robust and ready for mainnet Obscuro is | DataDog avg:system.uptime{*} | Average Testnet uptime over the last 4 weeks|99%|
+| Number of wallets connected to Obscuro Gateway| Good proxy for the number of active users. Straightforward to capture.| Datadog? |Number of daily connections|500|
+| Number of transactions| Typical guage of the amount of activity on testnet (even though it can be gamed).| Datadog? |Number of daily transactions|2000|
```

Comment:
We should add a line to capture the number of RPC requests and see how many we can handle without falling over

Reply:
Agreed. Presumably this is also an indicator of the amount of interaction.